syntax = "proto3";

// This must match your expected python import.
package mygrpc.v1;

service Greeter {
    rpc SayHello (HelloRequest) returns (HelloResponse) {}
    rpc SayHelloAgain (HelloRequest) returns (HelloResponse) {}
}

message HelloRequest {
    string name = 1;
}

message HelloResponse {
    string message = 1;
}